Integrations/integration-ATD.yml

```diff
@@ -1069,10 +1095,7 @@ script:
     - contextPath: IP.Address
       description: IP's relevant to the sample
       type: string
-    - contextPath: ATD.Task.IsComplete
-      description: Stating if the detonation was complete or not
-      type: string
-    description: Detonate file through McAfee ATD
+    description: Deprecated, please use detonate playbook instead.
```

Comment:
Remove `please` please :)
and if it's deprecated, then please add `deprecate: true`